src/Printers/ConsoleResultsPrinter.php

```diff
@@ -106,6 +110,21 @@ protected function renderSummaryTable(array $fileCounts): void
         $table->render();
     }
 
+    protected function printGithubAnnotationLines(array $results)
+    {
+        /** @var FileSearchResults $scanResult */
+        foreach ($results as $scanResult) {
+            foreach ($scanResult->results as $result) {
+                $this->output->writeln(sprintf(
+                    "::error file=%s,line=%d,col=%d::Found a ray call",
+                    str_replace(getcwd() . DIRECTORY_SEPARATOR, '', $result->file()->filename),
```

Comment:
I'm not sure if we can just use makeFilenameRelative(). I will test it tomorrow.